bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -18,9 +18,13 @@ User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(bat
         last_confirmation_date: user.created_at
       )
       num_users_updated += 1
+    rescue Exception => e
```

Comment:
In Ruby, it’s a [best practice]([
https://www.honeybadger.io/blog/ruby-exception-vs-standarderror-whats-the-difference/]) to rescue `StandardError and not its super class `Exception`
```suggestion
    rescue StandardError => e
```

Reply:
@sureshc Thanks will update.